ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ethernet-interfaces.j2

```diff
@@ -159,12 +159,16 @@
 {%             endif %}
 {%         endfor %}
 {%     endif %}
-{# Link Tracking Groups #}
+{# Link Tracking Groups and multicast #}
```

Comment:
While I understand the intend of saving another loop, it is confusing that you mix these two. Same for vlan interfaces.